tpp-app/src/home/Calendar.js

```diff
@@ -1,10 +1,41 @@
 import React from 'react';
-import { Text, View } from 'react-native';
+import { Text, View, StyleSheet } from 'react-native';
+import { CalendarList } from 'react-native-calendars';
 
-export default function Calendar () {
-    return (
-        <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
-            <Text>Calendar!</Text>
-        </View>
-    )
+
+export default () => {
+    return (<CalendarList
+        // Callback which gets executed when visible months change in scroll view. Default = undefined
+        onVisibleMonthsChange={(months) => {console.log('now these months are visible', months);}}
```

Comment:
Don't think the `console.log` is necessary anymore. Otherwise everything looks good and it works perfectly! :D